test/parallel/test-timers-clear-null-does-not-throw-error.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// This test makes sure clearing timers with
+// 'null' or no input does not throw error
+
+assert.doesNotThrow(() => { clearInterval(null);});
```

Comment:
These arrow functions would be a bit cleaner without the braces, e.g. `assert.doesNotThrow(() => clearInterval(null));`